en/docs/setup/migrating-to-5110.md

```diff
@@ -228,7 +228,7 @@ Once all the above prerequisites have been met, follow the instructions given be
         transformToSymmetric: "true"
         ```
     
-    Under each migrator's parameters, find the property value of **currentEncryptionAlgrithm** and ensure that it matches with the value of the `org.wso2.CipherTransformation` property found in the `<OLD_IS_HOME>/repository/conf/carbon.properties` file.
+    Under each migrator's parameters, find the property value of **currentEncryptionAlgorithm** and ensure that it matches with the value of the `org.wso2.CipherTransformation` property found in the `<OLD_IS_HOME>/repository/conf/carbon.properties` file.
```

Comment:
Can we make sure that this is the value read by the code too?

Reply:
It is, [1].

[1] https://github.com/wso2-extensions/identity-migration-resources/blob/4e25bed0924f353f30c2d07c1680aeaf7c9b4e41/components/org.wso2.is.migration/migration-resources/migration-config.yaml#L211